examples/FacebookTabsExample/FacebookTabBar.js

```diff
@@ -54,7 +55,6 @@ const FacebookTabBar = React.createClass({
             <Icon
               name={tab}
               size={30}
-              color={this.props.activeTab == i ? 'rgb(59,89,152)' : 'rgb(204,204,204)'}
```

Comment:
1. I think it's the root cause of icon color flashing: when re-rendering, icon color is set to `rgb(59,89,152)` at first place, because `props.activeTab` changed, then the animation begins, it's set to `rgb(204,204,204)` again, that's why it blinks
2. initial color is set in `componentDidMount `, `this.setAnimationValue({ value: this.props.activeTab, })`, so I think it safe to remove it.